lib/util.js

```diff
@@ -196,8 +196,9 @@ function format(f) {
   var lastPos = 0;
   for (i = 0; i < f.length - 1; i++) {
     if (f.charCodeAt(i) === 37) { // '%'
+      const nextChar = f.charCodeAt(++i);
       if (a !== arguments.length) {
```

Comment:
Would it make sense to cache `arguments.length` given the extensive usage throughout `format`?

Reply:
Actually, that might be worse for perf after reviewing how V8 handles it. You can ignore, I believe.